apps/dotcms-ui/src/app/portlets/dot-edit-page/content/services/dot-edit-content-html/dot-edit-content-html.service.spec.ts

```diff
@@ -773,6 +796,100 @@ describe('DotEditContentHtmlService', () => {
         });
     });
 
+    describe('inline editing', () => {
+        let dotWorkflowActionsFireService: DotWorkflowActionsFireService;
+        let dotGlobalMessageService: DotGlobalMessageService;
+
+        beforeEach(() => {
+            dotWorkflowActionsFireService = TestBed.inject(DotWorkflowActionsFireService);
+            dotGlobalMessageService = TestBed.inject(DotGlobalMessageService);
+        });
+
+        it('should call saveContentlet and save the content', () => {
+            spyOn(dotWorkflowActionsFireService, 'saveContentlet').and.returnValue(of({}));
+            const fakeElem: HTMLElement = fakeDocument.querySelector(
+                '[data-test-id="inline-edit-element-title"]'
+            );
+
+            service.contentletEvents$.next({
```

Comment:
can we test this doing the blur element directly from the element instead call the observable directly ?

Reply:
Yeah we could. Is it worth it?